Dockerfile

```diff
@@ -23,7 +23,10 @@ ARG V
 # Please do not add any dependency updates before the 'make install' here,
 # as that will mess with caching for incremental builds!
 #
-RUN make LOCKDEBUG=$LOCKDEBUG PKG_BUILD=1 V=$V SKIP_DOCS=true DESTDIR=/tmp/install clean-container build install
+RUN make LOCKDEBUG=$LOCKDEBUG PKG_BUILD=1 V=$V SKIP_DOCS=true DESTDIR=/tmp/install clean-container build install \
+    && rm /tmp/install/usr/bin/cilium-operator \
```

Comment:
Maybe we should rather customize Makefile to be able to build Cilium without those binaries?

Reply:
This would be a bit tidier, help to consolidate the build logic into a single place rather than spread across Makefiles and Dockerfiles.

Reply:
Done